lib/house.rb

```diff
@@ -0,0 +1,16 @@
+class House
+attr_reader :price, :address, :rooms
```

Comment:
This is a good start, but once again, if there are no tests for this, then in our eyes it's worthless. Make sure to write tests first!